base.js

```diff
@@ -6,10 +6,10 @@ module.exports = {
     './rules/style',
     './rules/variables',
     './rules/es6',
-    './rules/imports',
+    './rules/plugins/imports',
     './rules/strict',
-    './rules/flow.js',
-    './rules/eslint-comments',
+    './rules/plugins/flow',
+    // './rules/plugins/eslint-comments',
```

Comment:
Won't this be useful to remove unused comments?

Reply:
The plugin is not supported anymore. But I think we can use https://eslint.org/docs/latest/use/configure/rules#report-unused-eslint-disable-comments instead